apps/frontend/src/app/3_organisms/TransactionStepDialog/components/TransactionStep/TransactionStep.tsx

```diff
@@ -71,7 +74,8 @@ export const TransactionStep: FC<TransactionStepProps> = ({
             ? transaction.args[1]
             : undefined,
         gasPrice,
-        gasLimit: gasLimit.toString(),
+        //Increase gas limit by 20% to make sure tx won't fail for this reason
```

Comment:
```suggestion
        //TODO: replace with default gas limit - increase gas limit by 20% to make sure tx won't fail
```

Update comment here so we can find it easily later when we replace these with suggested default gas limits, and extract calculation to separate section (same thing for other instance)